src/main/g8/default.properties

```diff
@@ -1,3 +1,6 @@
-name=my-awesome-lambda
+name=Change Me
```

Comment:
I disagree as the name shouldn't carry any hint on the formatting, but that's really a detail